### ISSUE TITLE
GLM linear

### ISSUE THREAD

**Issue by OriolAbril** [pymc-devs/pymc-examples#81]
File: https://github.com/pymc-devs/pymc-examples/blob/main/examples/generalized_linear_models/GLM-linear.ipynb
Reviewers:

> The sections below may still be pending. If so, the issue is still available, it simply doesn't
> have specific guidance yet. Please refer to [this overview of updates](https://github.com/pymc-devs/pymc-examples/wiki/Notebook-updates-overview)

## Known changes needed
Changes listed in this section should all be done at some point in order to get this
notebook to a "Best Practices" state. However, these are probably not enough!
Make sure to thoroughly review the notebook and search for other updates.

### General updates
*

### ArviZ related
*

## Changes for discussion
Changes listed in this section are up for discussion, these are ideas on how to improve
the notebook but may not have a clear implementation, or fix some know issue only partially.

### General updates
*

### ArviZ related
*

**Comment by chiral-carbon** [comment]
Hello! I commented under issue #58  too but this issue is also one I'd like to take up and attempt to make the suggested improvements.

**Comment by OriolAbril** [comment]
I'll assign this to you too,  as PRs are only on a per notebook basis here, it's no problem to work on 2-3 notebooks simultaneously

**Comment by chiral-carbon** [comment]
great, thank you!

**Comment by chiral-carbon** [comment]
hello! had a plots related doubt. apart from using `figzise` as kwargs in `plot_trace` in cell 7 (using ArviZ), is there any other change that should be done? it's a very minor change so I wanted to confirm in case I might have misunderstood. 

the plot code looks like this currently:
'az.plot_trace(trace, figsize=(11,7));`

is this sufficient?

**Comment by OriolAbril** [comment]
Yes, the notebook is short and concise and there aren't many things to update

**Comment by chiral-carbon** [comment]
okay, got it. thanks.

**Comment by OriolAbril** [comment]
Needs to be updated to use bambi instead of glm module

**Comment by drbenvincent** [comment]
Link to notebook broken. Does this notebook exist any more?

**Comment by OriolAbril** [comment]
This was moved to https://docs.pymc.io/en/latest/learn/core_notebooks/GLM_linear.html. It has already been updated to v4 and is part of the subset being auto-executed every time we build docs
